torch/jit/frontend.py

```diff
@@ -285,6 +285,10 @@ def build_AnnAssign(ctx, stmt):
 
     @staticmethod
     def build_Delete(ctx, stmt):
+        if len(stmt.targets) > 1:
+            source_range = ctx.make_range(stmt.lineno, stmt.col_offset, stmt.col_offset + len("del"))
+            raise NotSupportedError(source_range,
+                                    'del with more than 1 operands are not supported')
```

Comment:
`del with more than one operand is not supported`

Reply:
Done.